### PR TITLE
add nicer empty content message

### PR DESCRIPTION
**Pull request by jancborchardt**
Adjust it to the proper empty content message, just like in Files. Please review @owncloud/designers @karlitschek @nickvergessen

### CODE REVIEW

**Comment by karlitschek** [comment]
sweeeeet :+1:

File: templates/stream.body.php [hunk 1]
```diff
@@ -31,8 +31,10 @@
 <?php $_['appNavigation']->printPage(); ?>
 
 <div id="app-content">
-	<div id="no_activities" class="hidden">
-		<div class="body"><?php p($l->t('You will see a list of events here when you start to use your %s.', $theme->getTitle())) ?></div>
+	<div id="emptycontent" class="hidden">
+		<div class="icon-activity"></div>
+		<h2><?php p($l->t('No activity yet')); ?></h2>
+		<p><?php p($l->t('This stream will show events like additions, changes & shares')); ?></p>
```
Comment (nickvergessen) [inline templates/stream.body.php]: *like >file< additions?

Comment (jancborchardt) [inline templates/stream.body.php]: In the future we want to show actions from apps as well I suppose, so I kept it generic.

**Comment by jancborchardt** [comment]
@nickvergessen homie ;)

**Comment by karlitschek** [comment]
can we merge this? :-)

**Comment by enoch85** [comment]
Looks nice! :+1: Not tested

**Comment by jancborchardt** [comment]
@nickvergessen please check and merge. Or someone from @owncloud/designers maybe?

**Comment by MorrisJobke** [comment]
Looks a lot nicer :+1:

**Comment by nickvergessen** [comment]
No we cant merge it, because it displays an incorrect message for filters.
Anyway will do a follow up PR

**Comment by nickvergessen** [comment]
Follow up #275

**Pull request merged**